docs/how-to/policy-guard/policy_guard.mdx

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 1
+title: Policy Guard
+---
+
+Policy Guards provide centralized control over permissions and roles across projects within an organization. This allows for consistent, organization-wide enforcement of rules regarding access and actions within projects.
+
+---
+
+## What is a Policy Guard?
+
+A Policy Guard defines specific permissions for roles across multiple projects.
+For example, if you need only "admin" roles to create documents, while "viewer" roles have read-only access, you can enforce these rules through a Policy Guard, ensuring that all projects included in this policy adhere to the same permissions consistently.
+
+## Key Capabilities of Policy Guards
+
+- **Centralized Control:** Manage permissions and roles at the organization level, allowing for streamlined rule-setting across projects.
+- **Flexible Role Management:** Define what each role can or cannot do across projects included in the policy.
+- **Project Inclusion and Exclusion:** Easily add or remove projects from a policy guard to control their permissions.
+- **Permission Rules:** Set specific permission rules, such as granting or revoking permissions for certain roles to perform actions on resources (e.g., creating, reading documents).
```

Comment:
Let's add here an important note ` :::note IMPORTANT ! ` that only a workspace Owner can add or remove new guarding policy rules.